store/postgres/src/block_range.rs

```diff
@@ -170,9 +174,9 @@ impl EntityBlockRange {
             }
```

Comment:
This could just push the bind param `block + 1`

Reply:
I wasn't able to make the compiler borrower happy...